jsplumb-persistence-plugin.js

```diff
@@ -1,207 +1,208 @@
+                  endpointArray.push([endpoint.type, options]);
+              }
+              console.log(options);
+          });
 
-            connections.push({
-                path: connector.getPath(),
-                segment: connector.getSegments(),
-                connectorType: type,
-                connectorAttr: attrs,
-                connectionId: connection[i].id,
-                sourceId: connection[i].sourceId,
-                targetId: connection[i].targetId,
-                sourceEndpointUuid: connection[i].endpoints[0].getUuid(),
-                targetEndpointUuid: connection[i].endpoints[1].getUuid(),
-                paintStyle: connection[i].getPaintStyle(),
-                endpointStyle: function()
-                {
-                    var temp = [];
-                    connection[i].endpoints.forEach(function(endpoint) {
-                        temp.push(endpoint.getPaintStyle());
-                    });
-                    return temp;
-                }(),
-                hoverPaintStyle: connection[i].getHoverPaintStyle(),
-                endpoint: endpointArray,
-                anchors: function()
-                {
-                    var temp = [];
-                    connection[i].endpoints.forEach(function(endpoint) {
-                        var tempObj = {
-                            uuid: endpoint.getUuid(),
-                            x: endpoint.anchor.x,
-                            y: endpoint.anchor.y,
-                            orientation: endpoint.anchor.orientation,
-                            offset: endpoint.anchor.offsets,
-                            parameters: endpoint.getParameters(),
-                            type: endpoint.anchor.type
-                        };
-                        temp.push(tempObj);
-                    });
-                    return temp;
-                }(),
-                labelText: connection[i].getLabel(),
-                overlays: $.map(connection[i].getOverlays(), function(overlay) {
-                    var temp = new Array();
-                    var obj = {};
-                    for (var key in overlay) {
-                        if (typeof overlay[key] !== 'function' && typeof overlay[key] !== 'object' && typeof overlay[key] != 'undefined')
-                        {
-                            if (key == 'loc')
-                            {
-                                obj["location"] = overlay[key];
-                            } else {
-                                obj[key] = overlay[key];
-                            }
-                        }
-                    }
-                    obj["cssClass"] = overlay.canvas.className;
-                    temp.push(obj);
-                    return temp;
-                })
-            });
-        }
+          connections.push({
+              // path: connector.getPath(), //No need cuz it causing error on saving
```

Comment:
path needs to be saved for the connections to load back with endpoints

Reply:
![Screenshot-20200630102159-406x96](https://user-images.githubusercontent.com/32875906/86102312-a3530880-babb-11ea-806f-641850d24983.png)
Gives me this error